Realm/Tests/RLMTestCase.m

```diff
@@ -198,11 +198,15 @@ - (void)waitForNotification:(NSString *)expectedNote realm:(RLMRealm *)realm blo
     [token stop];
 }
 
-- (void)dispatchAsync:(dispatch_block_t)block {
+- (dispatch_queue_t)bgQueue {
     if (!_bgQueue) {
         _bgQueue = dispatch_queue_create("test background queue", 0);
     }
-    dispatch_async(_bgQueue, ^{
+    return _bgQueue;
+}
+
+- (void)dispatchAsync:(dispatch_block_t)block {
```

Comment:
But `-dispatchAsyncAndWait:` calls `-dispatchAsync:` internally, ensuring that `_bgQueue` is initialized?